<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab</class>
 <widget class="QWidget" name="Tab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>690</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="placeholderText">
      <string>Paste URLs here...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="donationLayout">
     <item>
      <spacer name="horizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1166</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="donationLinkLabel">
       <property name="toolTip">
        <string>This app is the result of countless hours of development by Marko PejiÄ‡. If it's useful to you, please consider supporting its development with a donation.&lt;br&gt;Thank you!</string>
       </property>
       <property name="text">
        <string>&lt;a href=&quot;https://downloader.markopejic.com/donate&quot;&gt;Donate&lt;/a&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QFormLayout" name="downloadOptionsLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldGrowthPolicy::FieldsStayAtSizeHint</enum>
     </property>
     <property name="rowWrapPolicy">
      <enum>QFormLayout::RowWrapPolicy::WrapLongRows</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="formatLabel">
       <property name="text">
        <string>&lt;u&gt;F&lt;/u&gt;ormat:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="formatComboBox">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Show Options (Alt+F)</string>
       </property>
       <item>
        <property name="text">
         <string>Video</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Audio</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="qualityLabel">
       <property name="text">
        <string>&lt;u&gt;Q&lt;/u&gt;uality:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="qualityComboBox">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Show Options (Alt+Q)</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
       </property>
       <property name="placeholderText">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="subtitlesLabel">
       <property name="text">
        <string>&lt;u&gt;S&lt;/u&gt;ubtitles:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="subtitlesComboBox">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Show Options (Alt+S)</string>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
       </property>
       <property name="placeholderText">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="downloadFolderLabel">
       <property name="text">
        <string>Download Folder:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="downloadFolderLayout">
       <item>
        <widget class="QLabel" name="downloadFolderIndicatorLabel">
         <property name="text">
          <string>None</string>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setDownloadFolderButton">
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>Alt+C</string>
         </property>
         <property name="text">
          <string>&amp;Change</string>
         </property>
         <property name="icon">
          <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>697</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="cropThumbnailsCheckBoxLayout">
     <item>
      <spacer name="horizontalSpacer3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cropThumbnailsCheckBox">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Useful when downloading music.&lt;br&gt;(Alt+T)</string>
       </property>
       <property name="text">
        <string>Crop &amp;thumbnails / album arts to square shape</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer4">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="embedSubtitlesCheckBoxLayout">
     <item>
      <spacer name="horizontalSpacer5">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="embedSubtitlesCheckBox">
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Embeds subtitles instead of downloading them to a separate file.&lt;br&gt;(Alt+S)</string>
       </property>
       <property name="text">
        <string>&amp;Embed subtitles to videos</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer6">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer7">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>697</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actionButtonsLayout">
     <item>
      <spacer name="horizontalSpacer8">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="dataPullButton">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>32</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Pulls data from the URLs, allowing you to set a custom quality and subtitle language to download.&lt;br&gt;(Alt+P)</string>
       </property>
       <property name="text">
        <string>&amp;Pull Data</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downloadButton">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>32</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::FocusPolicy::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>Alt+D</string>
       </property>
       <property name="text">
        <string>&amp;Download</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::GoDown"/>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer9">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer10">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>697</width>
       <height>60</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="statusIndicatorsLayout">
     <item>
      <spacer name="horizontalSpacer11">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="statusIconLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer12">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>3</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="font">
        <font>
         <family>Cantarell</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer13">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
